src/app/common/stores/applications/SsvWeb/Validator.store.ts

```diff
@@ -577,8 +579,9 @@ class ValidatorStore extends BaseStore {
     const processStore: ProcessStore = this.getStore('Process');
     const okResponse = { id: 0, name: '', errorMessage: '' };
     const operatorNotExistResponse = { id: 1, name: 'operator_not_exist', errorMessage: 'Operators data incorrect, check operator data and re-generate keyshares.json.' };
-    const validatorExistResponse = { id: 3, name: 'validator_exit', errorMessage: 'Validator is already registered to the network, ', subErrorMessage:  'please try a different keystore file.' };
     const operatorNotMatchingResponse = { id: 2, name: 'operators_not_matching', errorMessage: 'Operators data does not match cluster,', subErrorMessage:  'check operator data and re-generate keyshares.json' };
+    const validatorExistResponse = { id: 3, name: 'validator_exit', errorMessage: 'Validator is already registered to the network, ', subErrorMessage:  'please try a different keystore file.' };
+    const checksumErrorResponse = { id: 4, name: 'ERROR', errorMessage: 'Invalid Ethereum address. Please check that you have entered the address correctly and try again.' };
```

Comment:
Use here the same approach with constants instead of raw values, like here:
https://github.com/bloxapp/ssv-web/pull/175#discussion_r1181183269

Reply:
done